src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -26,23 +26,32 @@
 package java.lang.invoke;
 
 import jdk.internal.misc.CDS;
-import jdk.internal.org.objectweb.asm.*;
 import jdk.internal.util.ClassFileDumper;
-import sun.invoke.util.BytecodeDescriptor;
 import sun.invoke.util.VerifyAccess;
 import sun.security.action.GetBooleanAction;
 
 import java.io.Serializable;
-import java.lang.constant.ConstantDescs;
+import java.lang.classfile.ClassBuilder;
+import java.lang.classfile.ClassFile;
+import static java.lang.classfile.ClassFile.*;
```

Comment:
Nit: group static imports in line 52 following this file convention.

Reply:
Fixed, thank you.